example/spec/parallel/moveGranules/MoveGranulesCollectionSpec.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+
+// TODO: rename this file to be clear it is also testing moving zero byte files
+
+/**
+ * LocalStack has a bug where it ignores the `Tagging` parameter to the
```

Comment:
Is this copypasta or sometihng new?

Reply:
this started life at a copy of MoveGranulesTagsSpec.js
I could probably afford to follow up on that TODO in the tags side and rearrange it in collection